UPGRADING.md

```diff
@@ -0,0 +1,105 @@
+# Upgrading Guide
+
+See the [CHANGELOG.md](./CHANGELOG.md) for detailed information about what has changed between versions.
+
+This guide is useful to figure out what you need to do between breaking changes.
+
+As always, [submit issues](https://github.com/FortAwesome/vue-fontawesome/issues/new) that you run into with this guide or with these upgrades to us.
+
+## 0.0.x to 0.1.0
+
+### Renamed packages
+
+The following packages have been renamed as part of 5.1.0 of Font Awesome.
+
+_All packages are in the [@fortawesome NPM scope](https://www.npmjs.com/search?q=scope:fortawesome&page=1&ranking=optimal)_
+
+| Old package(1)           | New package            |
+|--------------------------|------------------------|
+| fontawesome              | fontawesome-svg-core   |
+| fontawesome-free-solid   | free-solid-svg-icons   |
```

Comment:
It should be noted that the new packages are not yet available via yarn.

Reply:
@texdc you should be able to access the pre-releases with `yarn` or `npm`. They are tagged `prerelease`.

Try `yarn add @fortawesome/free-solid-svg-icons@5.1.0-6`

Reply:
I had to use `v5.1.0-3`, but close enough for jazz.  Thanks.